modules/apps/frontend-js/frontend-js-web/test/liferay/util/portlet_url/create_portlet_url.ts

```diff
@@ -12,34 +12,35 @@
  * details.
  */
 
-'use strict';
-
-import createPortletURL from '../../../../src/main/resources/META-INF/resources/liferay/util/portlet_url/create_portlet_url.es';
+import createPortletURL from '../../../../src/main/resources/META-INF/resources/liferay/util/portlet_url/create_portlet_url';
 
 describe('Liferay.Util.PortletURL.createPortletURL', () => {
 	afterEach(() => {
-		Liferay.ThemeDisplay.getPortalURL.mockRestore();
+		(Liferay.ThemeDisplay.getPortalURL as jest.Mock<
+			string,
+			[]
+		>).mockRestore();
```

Comment:
Maybe to avoid things like this for global objects we can create a base interface for Liferay with generic types to be able to use it in tests and another one for development.